app/avatar/models.py

```diff
@@ -82,14 +82,14 @@ def save(self, *args, force_insert=False, force_update=False, **kwargs):
         """Override the save to perform change comparison against PNG and SVG fields."""
         if (self.svg != self.__previous_svg) or (self.svg and not self.custom_avatar_png):
```

Comment:
alternatively we could just remove the if statement and update every single time